lm_eval/evaluator.py

```diff
@@ -117,6 +127,9 @@ def simple_evaluate(
                 continue
 
         config = task_obj._config
+        if config["output_type"] == "greedy_until" and gen_kwargs is not None:
+            config["generation_kwargs"].update(gen_kwargs)
```

Comment:
might raise errors when we conflict with a task’s provided generation_kwargs—how should we proceed in this case?

Reply:
I've added a warning in line 93. I think these feature would be intended to overwrite parameters originally set in each yaml.

```
        eval_logger.warning(
            f"generation_kwargs specified through cli, these settings will be used over set parameters in yaml tasks."
        )
```

Reply:
@haileyschoelkopf made changes to provide the above warning. Lmk if there is anything else. Other than that it should be good to go.